src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -86,10 +86,14 @@ unsafe fn configure_llvm(sess: &Session) {
 // array, leading to crashes.
 
 const ARM_WHITELIST: &[(&str, Option<&str>)] = &[
+    ("aclass", Some("arm_target_feature")),
```

Comment:
As for `mclass` and `rclass` too, this should be renamed to `a-profile` before stabilisation. IMO it's better to do that now to avoid any unstable code using it to have to be changed.

Reply:
Also, before stabilization, `neon` on aarch64 should be renamed to `asimd`. @japaric it might be worth it to open an issue to track these on stdsimd, there might already be an issue open for the `asimd` rename task, so maybe this could be added to that one.

Reply:
Hmm, how does stabilization work for target_features? `#[cfg(target_feature = "mclass")]` is *not* feature gated on stable but it doesn't work. AFAICT that feature gate will work in the next beta which comes out in a week *unless* there's some "this is a nightly only feature" logic that I didn't see.

Reply:
> Hmm, how does stabilization work for target_features? 

In the RFC for the intrinsics, you write which target features they need, and that's pretty much it. We can stabilize them with the intrinsics.